### ISSUE TITLE
Is it possible to see the solution when someone already answered it?

### ISSUE THREAD

**Issue by yonder17** [firasdib/Regex101#1810]
I started to do the quiz task and I really like it, the only thing that I would like you to do is that when we solved the task and show the statistics also show which solution is the best, all this with educational purposes. Thank you

**Comment by mtnbikerjoshua** [comment]
I disagree. I like that you can keep trying to improve your solution after seeing that someone else had a shorter one.

**Comment by KonnyK** [comment]
You could make it that you have to "finalize" your answer in order to be able to see other people's solutions. Until then you can only see how long their solutions. This way after really trying your best you can still learn. I've found myself wondering what I missed, but with no possible clue you just give up and go to the next one. Or you post it on stack overflow. ^^
